### DISCUSSION TITLE
[API] Request for adding imageTitle into response for recently played games

### DISCUSSION THREAD

**Discussion by DFelten** [RetroAchievements/RAWeb#2068] [Ideas]
I am currently working on an app for achievements and would like to be able to display not only the small icons, but also the titles in lists. To do this, I would currently have to start another request to get the game information.

Would it be possible to include the title image in the response of the game list?

Current repsonse:

```json
{
    {
        "GameID": 11310,
        "ConsoleID": 12,
        "ConsoleName": "PlayStation",
        "Title": "Wild Arms",
        "ImageIcon": "/Images/058584.png",
        "LastPlayed": "2023-11-26 13:07:58",
        "NumPossibleAchievements": 156,
        "PossibleScore": 1079,
        "NumAchieved": 26,
        "ScoreAchieved": 127,
        "NumAchievedHardcore": 26,
        "ScoreAchievedHardcore": 127
    },
}
```

This is currently missing for me:

```json
"ImageTitle": "/Images/019083.png",
```

**Comment by wescopeland** [comment]
https://github.com/RetroAchievements/RAWeb/pull/2069
